lib/linter.js

```diff
@@ -327,6 +327,22 @@ Linter.prototype.parseConfigName = function(name) {
     };
   }
 
+  if (name.toLowerCase().startsWith('plugin:@')) {
```

Comment:
Please add a low level spec / unit test to verify the correct parsing behavior.

Reply:
I've added the test.